var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -44,15 +42,46 @@ class Kokkos(Package):
     version('2.03.00', 'f205d659d4304747759fabfba32d43c3')
     version('2.02.15', 'de41e38f452a50bb03363c519fe20769')
     version('2.02.07', 'd5baeea70109249f7dca763074ffb202')
+    version('develop', git='https://github.com/kokkos/kokkos',
+            branch='develop')
 
+    variant('serial', default=True, description="enable Serial backend (default)")
     variant('qthreads', default=False, description="enable Qthreads backend")
     variant('cuda', default=False, description="enable Cuda backend")
     variant('openmp', default=False, description="enable OpenMP backend")
 
+    # Kokkos options
+    variant('aggressive_vectorization', default=False,
+            description="set aggressive_vectorization Kokkos option")
+    variant('disable_profiling', default=False,
+            description="set disable_profiling Kokkos option")
+    variant('disable_dualview_modify_check', default=False,
+            description="set disable_dualview_modify_check Kokkos option")
+    variant('enable_profile_load_print', default=False,
+            description="set enable_profile_load_print Kokkos option")
+    variant('compiler_warnings', default=False,
+            description="set compiler_warnings Kokkos option")
+    variant('disable_deprecated_code', default=False,
+            description="set disable_deprecated_code Kokkos option")
+    variant('enable_eti', default=False,
+            description="set enable_eti Kokkos option")
+
+    # CUDA options
+    variant('force_uvm', default=False,
+            description="set force_uvm Kokkos CUDA option")
+    variant('use_ldg', default=False,
+            description="set use_ldg Kokkos CUDA option")
+    variant('rdc', default=False,
+            description="set rdc Kokkos CUDA option")
+    variant('enable_lambda', default=False,
+            description="set enable_lambda Kokkos CUDA option")
+
     gpu_values = ('Kepler30', 'Kepler32', 'Kepler35', 'Kepler37',
                   'Maxwell50', 'Maxwell52', 'Maxwell53',
                   'Pascal60', 'Pascal61')
```

Comment:
is volta missing here?

Reply:
Yes! An oversight - corrected now. Thank you!